ci-scripts/unit-tests.sh

```diff
@@ -2,108 +2,33 @@
 set -e
 set -o pipefail
 
-echo "-----"
-
-echo "Running unit tests and code coverage for cds"
-
-npx nx test cds --no-watch --source-map --code-coverage --browsers ChromeHeadless
-
-echo "Running schematics unit tests and code coverage for CDS library"
-
-npm --prefix integration-libs/cds run test:schematics -- --coverage
-
-echo "Running unit tests and code coverage for product-configurator library"
-
-npx nx test product-configurator --source-map --no-watch --code-coverage --browsers ChromeHeadless
-
-echo "Running schematics unit tests and code coverage for product-configurator library"
-
-npm --prefix feature-libs/product-configurator run test:schematics -- --coverage
-
-echo "Running unit tests and code coverage for CDC"
-
-npx nx test cdc --source-map --no-watch --code-coverage --browsers ChromeHeadless
-
-echo "Running schematics unit tests and code coverage for cdc library"
-
-npm --prefix integration-libs/cdc run test:schematics -- --coverage
-
-echo "Running unit tests and code coverage for Digital-Payments"
-
-npx nx test digital-payments --source-map --no-watch --code-coverage --browsers ChromeHeadless
-
-echo "Running schematics unit tests and code coverage for Digital-Payments library"
-
-npm --prefix integration-libs/digital-payments run test:schematics -- --coverage
-
-echo "Running unit tests and code coverage for EPD Visualization"
-
-npx nx test epd-visualization --source-map --no-watch --code-coverage --browsers ChromeHeadless
-
-echo "Running schematics unit tests and code coverage for EPD Visualization library"
-
-npm --prefix integration-libs/epd-visualization run test:schematics -- --coverage
-
-echo "Running unit tests and code coverage for s4om"
-
-npx nx test s4om --source-map --no-watch --code-coverage --browsers ChromeHeadless
-
-echo "Running schematics unit tests and code coverage for s4om library"
-
-npm --prefix integration-libs/s4om run test:schematics -- --coverage
-
-echo "Running unit tests and code coverage for storefinder library"
+EXCLUDE_JEST=storefrontapp,storefrontstyles,schematics,setup
+EXCLUDE_INTEGRATION_LIBS=cdc,cds,digital-payments,epd-visualization,s4om
```

Comment:
We could reduce maintenance by gathering the names of libs in the integration folder, but this could be a future improvement.